### ISSUE TITLE
0.59.3 Scrolling Bar always visible

### ISSUE THREAD

**Issue by christianreiss** [RocketChat/Rocket.Chat#8895]
### Server Setup Information:

* Version of Rocket.Chat Server: 0.95.3
* Operating System: CentOS 7
* Deployment Method(snap/docker/tar/etc): Tar
* Number of Running Instances:  1
* DB Replicaset Oplog:
* Node Version: v4.8.4

### Steps to Reproduce:

- Always present when using (current) Firefox.
- Not present when using Chrome/Chromium

The Channel Scrollbar is always visible, even when nothing is there to scroll.

### Expected behavior:

As seen in Chrome:
![image](https://user-images.githubusercontent.com/7657348/32980727-48a2458a-cc6c-11e7-8af1-370db186787d.png)

### Actual behavior:

As seen in Firefox:
![image](https://user-images.githubusercontent.com/7657348/32980707-dd78c978-cc6b-11e7-80bc-28207cfa68cb.png)

**Comment by localguru** [comment]
same on demo.rocket.chat

**Comment by ArchimedesFM** [comment]
same with Internet Explorer 11.
Rocket.Chat 0.59.3

**Comment by Gummikavalier** [comment]
Found this workaround at open.rocket.chat (I recall):

Administration -> Layout -> Custom.CSS
```
/* Hide Firefox scrollbar in left pane if not needed */
.rooms-list { overflow-y: auto; }
```

**Comment by christianreiss** [comment]
I can confirm the workaround from @Gummikavalier works.

**Comment by ArchimedesFM** [comment]
The workaround works for Firefox, but not for InternetExplorer 11.

**Comment by Dremor** [comment]
Will this workaround be added to Rocket.Chat ? I'm using 0.64.2, and still viewing this.

**Comment by ggazzo** [comment]
closed by #10367
